cmd/contour/serve.go

```diff
@@ -170,37 +173,40 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 	}
 
 	// step 4. register our resource event handler with the k8s informers.
-	coreInformers.Core().V1().Services().Informer().AddEventHandler(eh)
+	informers := []cache.SharedIndexInformer{
```

Comment:
should these be passed through `registerEventHandler`?

Reply:
I have `informers` getting passed through now, is this in line with what you were suggesting?

Reply:
I wasn't able to find the place where those 6 informers added to `informers` had an event handler attached to them, but I think you resolved this in d8fb549